docker/ui/Dockerfile

```diff
@@ -3,9 +3,6 @@ FROM node:10
 # the setup looks way too complicated.  Let's stick with the official node:10 package even though
 # it's a large image
 
-# Expose Live-Reload port
-EXPOSE 35729
```

Comment:
Let's have this commented out as a TODO, since we don't have the time to make it work ATM